homeassistant/components/solaredge/__init__.py

```diff
@@ -1 +1,40 @@
 """The solaredge component."""
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DEFAULT_NAME, DOMAIN, CONF_SITE_ID
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+                vol.Required(CONF_API_KEY): cv.string,
+                vol.Required(CONF_SITE_ID): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup(hass, config):
+    """Platform setup, do nothing."""
+    hass.async_create_task(
```

Comment:
Check if there's a section for this domain in the config, if not just return True.